### PR TITLE
Added #playing? And #length for VoiceBot

### PR DESCRIPTION
**Pull request by SnazzyPine25**

### CODE REVIEW

**Comment by meew0** [comment]
What exactly would `length` be useful for? It's an internal detail for which I can't imagine a valid use case.

**Comment by SnazzyPine25** [comment]
Stream time and length can determine a good percentage of the song that's played, that's all I got. Honestly I just saw length being used and jumped to it

**Comment by meew0** [comment]
The length is slightly misleadingly named, it's not the length of the song but rather the length the VoiceBot has to wait after having sent one packet. This is a detail that is useless to expose to the public.

**Comment by SnazzyPine25** [comment]
Alright, should I delete that?

**Comment by meew0** [comment]
Yes please, as it will likely mislead people looking at the docs

File: lib/discordrb/voice/voice_bot.rb [hunk 1]
```diff
@@ -120,6 +121,14 @@ def pause
       @paused = true
     end
 
+    # @see #playing=
```
Comment (meew0) [inline lib/discordrb/voice/voice_bot.rb]: What method is this referencing? There is no `playing=` method in `VoiceBot`

**Pull request merged**